recipes/blosc/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.9.2" %}
+
+package:
+  name: blosc
+  version: {{ version }}
+
+source:
+  url: https://github.com/Blosc/c-blosc/archive/v{{ version }}.tar.gz
+  sha256: 6349ab927705a451439b2e23ec5c3473f6b7e444e6d4aafaff76b789713e9fee
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - cmake
+    - zlib
```

Comment:
In `conda-forge` we pin some `key` packages to ensure compatibility of the stack. Please change `zlib` to `zlib 1.2.*` here.(The pinnings can be found in https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies)